docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala

```diff
@@ -4,6 +4,7 @@
 
 package docs.http.scaladsl.server.directives
 
+import akka.http.scaladsl.model.HttpMethods
```

Comment:
Usually is better to have the imports within the examples, so they can be copy-pasted and run without needing to guess the right import.

Reply:
@jlprat you are absolutely right. In this case the import is unused. The first implementation checked if the Request was an HTTP GET. Forgot the remove it.